layers/cmd/cmdutil/csv.go

```diff
@@ -60,3 +60,39 @@ func ReadSpreadsheetFile(fileName string) ([][][]string, error) {
 	}
 	return ret, nil
 }
+
+func ReadSpreadsheetFilev2(fileName string) (map[string][][]string, error) {
```

Comment:
Ugly name. Make this ReadSpreadsheetFile, rename the original to ReadSheets, and call this one